### ISSUE TITLE
Enable hideDoubleBrackets plugin option from gatsby-theme-brain

### ISSUE THREAD

**Issue by saundersbd** [aravindballa/gatsby-theme-andy#2]
I noticed that the current version of gatsby-theme-brain allows you to hide the double bracket styling of internal links. It would be useful if I could enable this plugin option within gatsby-theme-andy.

**Comment by thisizkp** [comment]
I tried passing the options to the theme, even then it doesn't seem to be working. The theme does seem like reading the options and passing it to the gatsby-brain. Not sure where it's gone wrong.

```js
module.exports = {
  siteMetadata: {
    title: `Kp's notes`,
  },
  plugins: [
    {
      resolve: `gatsby-theme-andy`,
      options: {
        hideDoubleBrackets: true,
      },
    },
  ],
}
```

**Comment by aravindballa** [comment]
That is how you should be doing. However, the issue here is that I didn’t update to the latest `gatsby-theme-brain`. So, it doesn’t respect that option.

I will fix this soon.

**Comment by aravindballa** [comment]
Fixed in `v0.1.4`
